### PR TITLE
Handle ctrl click on links

### PR DESCRIPTION
**Pull request by lagartoverde**
Needs testing on windows and linux, I added the meta key so it works as expected on mac

### CODE REVIEW

**Review by elpoelma — APPROVED** [review]
Seems to work well!

File: addon/plugins/links/index.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      view: RdfaEditorView,
+      pos: number,
+      node: PNode,
+      nodePos: number,
+      event: MouseEvent
+    ) {
+      const schema = view.state.schema;
+      // The handler only handles clicks on nodes, and the link is a mark of the textnode inside the node, so we have to get it
+      const textNode = node.childAfter(pos);
+      if (!textNode || !textNode.node) return;
```
Comment (abeforgit) [inline addon/plugins/links/index.ts:14]: I know I know it's pedantic, and if you disagree you can merge without applying in this case (cause its really _just_ a return)

```suggestion
      if (!textNode || !textNode.node) { 
        return; 
      }
```

**Review by abeforgit — CHANGES_REQUESTED** [review]
Please add a changelog entry, then you can merge

File: addon/plugins/links/index.ts (hunk 1 above)
Comment (lagartoverde) [reply addon/plugins/links/index.ts]: I prefer your syntax, I don't know what happened :), thanks for catching that

**Pull request merged**